test/index.ts

```diff
@@ -13,10 +13,23 @@ export type Test = () => void;
 
 export { getCanisterActor } from './get_canister_actor';
 
+export interface RunTestsOptions {
```

Comment:
Is this abstraction necessary? Just to save 5 seconds?

Reply:
And if we're going to do such a big change, wouldn't we want to put cwd and tests inside of this options object?

Reply:
I think we should get rid of this for now.

Reply:
done